index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <a href="#" id="logo">LOGO</a>
+    <nav>
+      <ul id="nav-menu">
+        <li class="nav-link"><a href="#works">Portfolio</a></li>
+        <li class="nav-link"><a href="#about-myself">Myself</a></li>
+        <li class="nav-link"><a href="#form-section">Contact</a></li>
+      </ul>
+      <div id="mobile-navbar">
+        <div id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <ul id="mobile-nav-menu">
+          <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
```

Comment:
Great job @chuaindia 🤝

- [ ] Kindly style your UI properly. At the moment I can't identify which Figma template you chose.
- Your design
![Your design](https://user-images.githubusercontent.com/63917725/185608150-e169b2b1-d2de-4d65-bc27-18cfc627607c.PNG)